pilot/pkg/bootstrap/server.go

```diff
@@ -397,6 +400,12 @@ func (s *Server) Start(stop <-chan struct{}) error {
 		}()
 	}
 
+	if s.XDSServer.XEventing != nil {
+		s.XDSServer.XEventing.IstiodEvent("istio-system",
+			"istio-start",
```

Comment:
since this will become an API we should probably have some common constants somewhere with explanation of what they are.

Do events typically follow a namespaced format like `istio.io/start`?